mpas_analysis/ocean/plot_climatology_map_subtask.py

```diff
@@ -0,0 +1,424 @@
+
+        groupLink : str
+            a short name (with no spaces) for the link to the gallery group
+
+        galleryName : str
+            the name of the gallery in which this plot belongs
+
+        Authors
+        -------
+        Xylar Asay-Davis
+        """
+        self.outFileLabel = outFileLabel
+        self.fieldNameInTitle = fieldNameInTitle
+        self.mpasFieldName = mpasFieldName
+        self.obsFieldName = obsFieldName
+        self.observationTitleLabel = observationTitleLabel
+        self.unitsLabel = unitsLabel
+
+        # xml/html related variables
+        self.imageCaption = imageCaption
+        self.galleryGroup = galleryGroup
+        self.groupSubtitle = groupSubtitle
+        self.groupLink = groupLink
+        self.galleryName = galleryName
+        # }}}
+
+    def setup_and_check(self):  # {{{
+        """
+        Perform steps to set up the analysis and check for errors in the setup.
+
+        Authors
+        -------
+        Xylar Asay-Davis
+        """
+        # first, call setup_and_check from the base class (AnalysisTask),
+        # which will perform some common setup, including storing:
+        #     self.runDirectory , self.historyDirectory, self.plotsDirectory,
+        #     self.namelist, self.runStreams, self.historyStreams,
+        #     self.calendar
+        super(PlotClimatologyMapSubtask, self).setup_and_check()
+
+        config = self.config
+        self.startYear = config.getint('climatology', 'startYear')
+        self.endYear = config.getint('climatology', 'endYear')
+        self.startDate = config.get('climatology', 'startDate')
+        self.endDate = config.get('climatology', 'endDate')
+
+        mainRunName = config.get('runs', 'mainRunName')
+
+        self.xmlFileNames = []
+        self.filePrefixes = {}
+
+        if self.comparisonGridName == 'latlon':
+            self.filePrefix = '{}_{}_{}_years{:04d}-{:04d}'.format(
+                    self.outFileLabel, mainRunName,
+                    self.season, self.startYear, self.endYear)
+        else:
+            self.filePrefix = '{}_{}_{}_{}_years{:04d}-{:04d}'.format(
+                    self.outFileLabel, self.comparisonGridName,
+                    mainRunName, self.season, self.startYear, self.endYear)
+        self.xmlFileNames.append('{}/{}.xml'.format(self.plotsDirectory,
+                                                    self.filePrefix))
+        # }}}
+
+    def run_task(self):  # {{{
+        """
+        Plots a comparison of ACME/MPAS output to SST, MLD or SSS observations
+
+        Authors
+        -------
+        Luke Van Roekel, Xylar Asay-Davis, Milena Veneziani
+        """
+
+        season = self.season
+        comparisonGridName = self.comparisonGridName
+        self.logger.info("\nPlotting 2-d maps of {} climatologies for {} on "
+                         "the {} grid...".format(self.fieldNameInTitle,
+                                                 season, comparisonGridName))
```

Comment:
ah, here is the repeated logging statement (above). It's here and then again below, in `_plot_lat_lon` (not in `_plot_antarctic`). So I guess I would remove it from `_plot_lat_lon`.